test/Microsoft.Data.Sqlite.Tests/SqliteCommandTest.cs

```diff
@@ -171,6 +171,63 @@ public void Prepare_throws_when_command_text_contains_dependent_commands()
         }
 
         [Fact]
+        public void Multiple_command_executes_works()
+            => Multiple_command_executes_works((SqliteCommand command) =>
+            {
+                command.CommandText = string.Empty;
+                command.CommandText = "INSERT INTO Data (Value) VALUES (@value);";
+            });
+
+        [Fact]
+        public void Multiple_command_executes_works_2()
+            => Multiple_command_executes_works((SqliteCommand command) =>
+            {
+                command.CommandText = "INSERT INTO Data (Value) VALUES (@value);";
+            });
+
+        [Fact]
+        public void Multiple_command_executes_works_3()
+            => Multiple_command_executes_works((SqliteCommand command) => { });
```

Comment:
I think just this variation is needed. The other two aren't really testing anything related to the parameter binding.